packages/loader/container-loader/src/container.ts

```diff
@@ -982,7 +956,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         else if (this._attachState !== AttachState.Attached) {
             throw new UsageError(`The Container is not attached and cannot be connected`);
         }
-        else if (!this.connected) {
+        else {
```

Comment:
Removed this check since it was causing issues in tests where op processing is paused manually (without using `connect()`/`disconnect()`). As far as I can tell this check isn't really required and shouldn't have any harmful side effects.